assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -7707,52 +7787,130 @@ tests:
           model
             schema 1.1
           type user
-
-          type document
+          type resource
```

Comment:
Can we remove this test altogether? Seems like a duplicate of `resolution_too_complex_throws_error`

Reply:
`err_and_err_return_err` and `list_objects_with_subcheck_resolution_depth_exceeded` look identical to me.

<img width="1198" alt="image" src="https://github.com/user-attachments/assets/1c794643-d885-4eba-9da0-0fc3b3f7d24d">

 Let's keep `err_and_err_return_err` 

But let's keep `resolution_too_complex_throws_error`. This one's solution is one path that leads to an error, whereas `err_and_err_return_err`  has two paths that both lead to an error

Reply:
To be clear, remove: `list_objects_with_subcheck_resolution_depth_exceeded` and keep: `resolution_too_complex_throws_error`, and `err_and_err_return_err`, yes?

Reply:
Yes